go.mod

```diff
@@ -277,6 +277,7 @@ require (
 	github.com/secure-systems-lab/go-securesystemslib v0.8.0 // indirect
 	github.com/segmentio/asm v1.2.0 // indirect
 	github.com/segmentio/ksuid v1.0.4 // indirect
+	github.com/sethvargo/go-retry v0.2.4 // indirect
```

Comment:
Instead of introducing a new dependency, we should use the "github.com/spiffe/spire/pkg/agent/common/backoff" package that we already have

Reply:
I see. I will prepare a "final" version. Thank you for the comments.